conversation/ShareJS
# Upgrade uglifier

This upgrades uglifier while keeping the config as close as possible to the old version we were previously using.

To test, apply this diff:
```diff
diff --git a/examples/index.html b/examples/index.html
index 65031b0..54dc16b 100644
--- a/examples/index.html
+++ b/examples/index.html
@@ -195,7 +195,7 @@ You’re writing a web app. Your app contains data that users edit. Your users s
 
 <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.6.4/jquery.min.js"></script>
 <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.8.17/jquery-ui.min.js"></script>
-<script src="/share/share.uncompressed.js"></script>
+<script src="/share/share.js"></script>
 <script src="/share/textarea.js"></script>
 <script src="/lib/prettify.js"></script>
 <script src="/share/json.js"></script>
```

Then run `./bin/exampleserver` and ensure it still connects/lets you edit documents etc
